src/seeds/class.js

```diff
@@ -1 +1,114 @@
-// seeds here
+import mongoose from 'mongoose';
+
+export default [
+  {
+    _id: new mongoose.Types.ObjectId('64667748fc13ae7f027543cb'),
+    hour: '2:11',
+    day: '04/03/2023',
```

Comment:
```suggestion
    day: 'Monday',
```
The 'day' field should represent a day of the week, not a date. Please review the 'Class' model.
`day : "Monday`,